pkgs/by-name/mr/mricron/package.nix

```diff
@@ -0,0 +1,61 @@
+{ lib, stdenv, fetchurl, makeWrapper, unzip, autoPatchelfHook, makeDesktopItem, copyDesktopItems
+, gtk2, glib, gdk-pixbuf, pango, cairo, atk
+, freetype, fontconfig, xorg, zlib
+}:
+
+let
+  pkgDescription = "A application to display NIfTI medical imaging data.";
+in
+
+stdenv.mkDerivation rec {
+  pname = "mricron";
+  version = "1.0.20190902";
+
+  src = fetchurl {
+    url = "https://github.com/neurolabusc/MRIcron/releases/download/v${version}/MRIcron_linux.zip";
+    sha256 = "sha256-C155u9dvYEyWRfTv3KNQFI6aMWIAjgvdSIqMuYVIOQA=";
+  };
```

Comment:
Do we need `zip` or could we use `fetchFromGitHub`?

Reply:
I tried building it from source, but the compiler it is made for (lazarus) is hard to setup headless. 

It is a built binary zip in the releases, so i though fetch from zip is just as stable?

Reply:
gotcha. This should be just as stable. Could you then please add `lib.sourceTypes.binaryNativeCode` to `meta.sourceProvenance` (See https://nixos.org/manual/nixpkgs/unstable/#lib.sourceTypes.binaryNativeCode)

Reply:
Sure.